migration-from-4x.md

```diff
@@ -413,23 +457,23 @@ Dialogs are still opened via the `editor.windowManager.open(config)` api, howeve
 | `onpostrender` | `onpostrender` has been removed for the dialog configuration. The dialog configuration now includes an `initialData` setting for providing the initial state and an API to fetch or update the data. Refer to [this]({{site.baseurl}}/ui-components/dialog/#interactiveexampleusingredialconfigvoid) section for more information on how to configure `initialData`. |
 | [`url`]({{site.url}}/docs-4x/advanced/creating-custom-dialogs/) | URL dialogs now have their own API. For more information, see [`Custom URL dialogs`](#customurldialogs). |
 
-##### New settings:
+##### New dialog settings:
 
 | **New setting** | **Description** |
 | --------------- | --------------- |
 | [`initialData`]({{site.baseurl}}/ui-components/dialog/#configurationoptions) | An object containing the initial value for the dialog components. |
 | [`onCancel`]({{site.baseurl}}/ui-components/dialog/#configurationoptions) | A callback that is called when the dialog is cancelled without submitting any changes. |
 | [`onTabChange`]({{site.baseurl}}/ui-components/dialog/#configurationoptions) | A callback that is called when switching tabs in a [TabPanel]({{site.baseurl}}/ui-components/dialogcomponents/#tabpanel). |
 
-##### Changed settings:
+##### Changed dialog settings:
 
 | **Old setting** | **New setting** | **Description** |
 | --------------- | --------------- | --------------- |
 | `onchange` | [`onChange`]({{site.baseurl}}/ui-components/dialog/#configurationoptions) | `onchange` now takes a callback function which is passed an API helper function and data. |
 
-> Note: `onchange`, a callback that was previously used to detect changes on each component, has been removed. The changes are now tracked by a single callback function with the change data. Refer to the [Interactive example using redial(config): void]({{site.baseurl}}/ui-components/dialog/#interactiveexampleusingredialconfigvoid) for more information.
+> Note: The `onchange` callback has been removed. Changes are now tracked using the redial callback function. For information on using redial, see: [Interactive example using redial]({{site.baseurl}}/ui-components/dialog/#interactiveexampleusingredial).
```

Comment:
This has changed the original meaning and isn't quite correct now (granted the original was confusing). Firstly, there is no such thing as a redial callback function. So the main difference between 4 and 5 was that in 4 there was a `onchange` function attached to each component, however in 5 there's only a single `onChange` callback at the root of the dialog spec, similar to `onAction`. The redial example linked contains an example to that, but it's more so about using the `redial` API to change the components/options in the dialog.